examples/connext_dds/compression/c++/compression.idl

```diff
@@ -0,0 +1,15 @@
+/*
+ * (c) 2013-2021 Copyright, Real-Time Innovations, Inc.  All rights reserved.
+ *
+ * RTI grants Licensee a license to use, modify, compile, and create derivative
+ * works of the Software.  Licensee has the right to distribute object form
+ * only for use with RTI products.  The Software is provided "as is", with no
+ * warranty of any type, including any warranty for fitness for any purpose.
+ * RTI is under no obligation to maintain or support the Software.  RTI shall
+ * not be liable for any incidental or consequential damages arising out of the
+ * use or inability to use the software.
+ */
+
+struct string_1k {
+    string str;
+};
```

Comment:
This might lead to the creation of weird type names when you declare an object of string_1k. You could call it:

```
struct String1k {
    string str;
};
```

Reply:
I will change it to `StringLine` since the size would vary depends on the input file